packages/amplify/amplify_flutter/lib/src/hybrid_impl.dart

```diff
@@ -16,6 +16,7 @@ import 'dart:convert';
 
 import 'package:amplify_flutter/amplify_flutter.dart';
 import 'package:amplify_flutter/src/amplify_impl.dart';
+import 'package:flutter/material.dart';
```

Comment:
Why do we need the material library here? It provides only the UI widgets correct?

Reply:
good catch, removed. I think it was left from a previous implementation that used `describeEnum`.